docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
+
+orgs directory
+^^^^^^^^^^^^^^
 
-The ``.json`` files found in the ``orgs`` directory define the Salesforce DX org configurations that are available to the project. We cover scratch org management in depth in TODO: link Scratch Org Management. 
+The ``.json`` files found in the ``orgs`` directory define the Salesforce DX org configurations that are available to the project. We cover scratch org management in depth in [TODO: link Scratch Org Management].
 
-Datasets
+datasets
 ^^^^^^^^
 
-Each project may have one or more *datasets*: on-disk representations of record data that can be inserted into Salesforce orgs, and which can also be modified and re-captured during the evolution of the project. Datasets are stored in the ``datasets`` directory. Learn more about datasets in TODO: link Automating Data Operations.
+Each project may have one or more **datasets**: on-disk representations of record data that can be inserted into Salesforce orgs, and which can also be modified and re-captured during the evolution of the project. Datasets are stored in the ``datasets`` directory. Learn more about datasets in [TODO: link Automating Data Operations].
 
-Robot Framework Browser Automation
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+robot
+^^^^^
 
-CumulusCI integrates Robot Framework browser automation for end-to-end testing. Each project contains a ``robot`` directory, which stores the resources and test suites for the project's Robot Framework test suites. New CumulusCI projects start with a simple Robot test case that creates a Contact record. 
+Robot Framework provides browser automation for end-to-end testing. Each project contains a ``robot`` directory, which stores the project's Robot Framework test suites. New CumulusCI projects start with a simple Robot test case that creates a Contact record.
 
-While Robot Framework is used primarily for browser automation testing, in a small number of situations, Robot Framework may also be harnessed to help configure orgs where other strategies and APIs are insufficient.
+While Robot Framework is used primarily for browser automation testing, it can also be harnessed to help configure orgs where other strategies and APIs are insufficient. [TODO LINK TO EXPLAIN THIS FURTHER?]
 
-Unpackaged Metadata
+unpackaged metadata
 ^^^^^^^^^^^^^^^^^^^
 
-In the Product Delivery Model, a product doesn't just encompass the contents of a managed package or a single deployment. It also includes *unpackaged metadata*: extra bundles of Salesforce metadata that further tailor an org or complete the product.
+A product doesn't just encompass the contents of a managed package or a single deployment. It also includes **unpackaged metadata**: extra bundles of Salesforce metadata that further tailor an org or complete the product.
```

Comment:
```suggestion
A product doesn't just include the contents of a managed package or a single deployment. It also includes **unpackaged metadata**: extra bundles of Salesforce metadata that further tailor an org or complete the product.
```
Slightly simpler.

Reply:
Yeah, but then we're using "include" as the verb twice in a row. I think I chose encompass as shorthand for "that's not all that the project contains".

Reply:
Sometimes, echoing the verb can be a desirable, helpful device that ties the second idea closely to the first. When I first read it, "include" seemed like a simpler word choice than "encompass," as well as less cognitive load--I've already processed that verb the first time. But I respect your point that "encompass" might more precisely convey what you mean. In which case, never mind my edit! Just note that repetition isn't always a bad thing.